packages/core/src/util/mock-s3.ts

```diff
@@ -0,0 +1,38 @@
+import { S3Utils } from "../external/aws/s3";
+
+export class MockS3Utils extends S3Utils {
+  override async uploadFile({
+    bucket,
+    key,
+    //eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
why accept these if they aren't used?

Reply:
its extending the class and needs to have the same functions args